examples/machine_translation/model.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers import TransformerDecoder
+from keras_nlp.layers import TransformerEncoder
+
+
+class PositionalEmbedding(keras.layers.Layer):
```

Comment:
We should directly add this class to the API. It is very simply. We just need to settle on the signature (it should match the signature of the `Embedding` layer).

Reply:
I believe Fabien is looking at the design of positional embedding, so let's now keep this in the example, and later replace it?